### ISSUE TITLE
Check Python functionality is correct

### ISSUE THREAD

**Issue by hsauro** [sys-bio/roadrunner#104]
Someone reported that the following syntax did not work, check whether this is true or not.

rr.selections = ['time', 'S1']
result = rr.simulate (0, 10, 100)

**Comment by kirichoi** [comment]
It seems to work well. Are there a specific code/error code related to it?

**Comment by hsauro** [comment]
I don't have any more info, but ask Lucian, he reported from Germany.

**Comment by luciansmith** [comment]
Well, it was via Mike Hucka.  Here's what I had emailed him:

---
By default, it will give you the floating species, right.  If you want to set it yourself, you use 'selections':

r.selections = ['time', 'S1', 'S2, 'p1', 'k']
---

And here's what he emailed me back:

---
I couldn't seem to get the r.selections approach to work, but I did get the approach using

  result = r.simulate (0, 100, 100, ['time', 'S1', 'S2'])

to work as expected.
---

I'll ask him if he has any other details, or if he can maybe try again?

-Lucian
